include/seqan3/alphabet/concept.hpp

```diff
@@ -14,12 +14,29 @@
 
 #include <seqan3/alphabet/exception.hpp>
 #include <seqan3/core/concept/cereal.hpp>
-#include <seqan3/core/concept/core_language.hpp>
 #include <seqan3/core/detail/customisation_point.hpp>
-#include <seqan3/core/detail/reflection.hpp>
-#include <seqan3/core/type_traits/basic.hpp>
-#include <seqan3/core/type_traits/function.hpp>
-#include <seqan3/std/concepts>
+#include <seqan3/std/type_traits>
+
+// ============================================================================
+// obscure gcc 7, 8, 9 bug to get ADL working
```

Comment:
Is there a bug report for this?
And why would including small_vector fix this?

Reply:
I feel so stupid. This actually is what will be fixed by https://github.com/seqan/seqan3/pull/1228